Neo4j.Driver/Neo4j.Driver.IntegrationTests/ExamplesAsync.cs

```diff
@@ -509,12 +501,7 @@ public ResultConsumeExample(ITestOutputHelper output, StandAloneIntegrationTestF
                     {
                         var result = await tx.RunAsync("MATCH (a:Person) RETURN a.name ORDER BY a.name");
 
-                        List<string> list = new List<string>();
-
-                        while (await result.ReadAsync())
-                            list.Add(result.Current()[0].As<string>());
-
-                        return list;
+                        return await result.ToListAsync(r => r[0].As<string>());
```

Comment:
This code shows a "normal" usecase of `Task<List<T>> ToListAsync(..., Func<IRecord, T>)`